src/collections/Form/index.d.ts

```diff
@@ -173,7 +173,7 @@ interface FormGroupProps {
 
 export const FormGroup: React.ComponentClass<FormGroupProps>;
 
-interface FormInputProps extends InputProps, ReactFormEvents<HTMLInputElement>, ReactFocusEvents<HTMLInputElement> {
+interface FormInputProps extends InputBaseProps, ReactFormEvents<HTMLInputElement>, ReactFocusEvents<HTMLInputElement> {
```

Comment:
The signature for this:
```jsx
<Form.Input onChange={ ... } />
```
Should be identical to this:
```jsx
<Input onChange={ ...  } />
```
Wihch is:
```ts
onChange(event: React.SyntheticEvent<HTMLInputElement>, data: InputOnChangeData) => void
```

So the `onChange` signature from `ReactFormEvents<HTMLInputElement>` is not correct for this case since it does not include the `data` argument.  I'm not a ts user so I'm not exactly sure of the fix here, however, I wanted to note the actual signatures.